mmtrack/datasets/reid_dataset.py

```diff
@@ -0,0 +1,212 @@
+import copy
+from collections import defaultdict
+
+import numpy as np
+import torch
+from mmcls.datasets import DATASETS, BaseDataset
+
+
+@DATASETS.register_module()
+class ReIDDataset(BaseDataset):
+    """Dataset for ReID Dataset.
+
+    Args:
+        load_as_video (bool): If True, using COCOVID class
```

Comment:
`load_as_video` seems useless here. We should remove it from this function. If there are some conflicts with former build functions, adapt the building function

Reply:
`load_as_video` is used in `eval_hooks`.
https://github.com/open-mmlab/mmtracking/blob/master/mmtrack/core/evaluation/eval_hooks.py#L14